src/sync/installation.ts

```diff
@@ -176,16 +171,13 @@ export const isNotFoundError = (
 	const isNotFoundError = isNotFoundErrorType?.length > 0 || err?.status === 404;
 
 	isNotFoundError &&
-	logger.info(
-		{ job, task: nextTask },
-		"Repository deleted after discovery, skipping initial sync"
-	);
+	logger.info("Repository deleted after discovery, skipping initial sync");
 
 	return isNotFoundError;
 };
 
 // TODO: type queues
-async function doProcessInstallation(app, job, installationId: number, jiraHost: string, logger: LoggerWithTarget, scheduleNextTask: (delayMs) => void): Promise<void> {
+async function doProcessInstallation(app, data: BackfillMessagePayload, sentry: Hub, installationId: number, jiraHost: string, logger: LoggerWithTarget, scheduleNextTask: (delayMs) => void): Promise<void> {
```

Comment:
That's the whole thing which was done in this file: 

Getting rid of Job type parameter, and replacing it with "data" and "sentry" - only those properties of Job which are used in this method.

Also got rid of "task" parameter in logs everywhere except "Starting task" log